caiman/source_extraction/cnmf/utilities.py

```diff
@@ -172,14 +324,14 @@ def peak_local_max(image, min_distance=1, threshold_abs=None,
 
     # Non maximum filter
     if footprint is not None:
-        # image_max = ndi.maximum_filter(image, footprint=footprint,
-        #                                mode='constant')
-        image_max = cv2.dilate(image, footprint=footprint, iterations=1)
+        image_max = ndi.maximum_filter(image, footprint=footprint,
+                                       mode='constant')
+        # image_max = cv2.dilate(image, footprint=footprint, iterations=1)
     else:
         size = 2 * min_distance + 1
-        # image_max = ndi.maximum_filter(image, size=size, mode='constant')
-        image_max = cv2.dilate(image, cv2.getStructuringElement(
-            cv2.MORPH_RECT, (size, size)), iterations=1)
+        image_max = maximum_filter(image, size=size, mode='constant')
+        # image_max = cv2.dilate(image, cv2.getStructuringElement(
```

Comment:
We may just want to remove the commented-out code rather than update it?

Reply:
remove it